docs/source/analyze/weave-report.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Weave Report
+description: Analyze weave reports
```

Comment:
```suggestion
description: Analyze Weave reports
```
I think Weave should be capitalized; also, is just Weave okay vs Weave Net (did a bit of digging and it looked like we may be using their CNI plugin?)